### PR TITLE
fix(brain.fm): update querySelectors of my.brain.fm

### LINKED ISSUE #7373: brain.fm | my.brain.fm doesn't show detailed presence information
### Steps to Reproduce

1. Go to https://my.brain.fm/ (you need to be signed in)
2. "Choosing mode" is not shown in presence
3. Go to https://my.brain.fm and choose a mode
4. Song details and activity isn't shown in presence

### Expected Behavior

See song info and activity in Discord presence

### Actual Behavior

Generic presence info is shown

### Page URL

https://my.brain.fm/

### Screenshots

image

### Operating System

Debian 12

### Browser and Version

Chrome 113.0.5672.126

### Additional Comments

After looking through the source code, a lot of querySelectors are no longer valid. I will try to fix them in a pull request.

### PR DESCRIPTION
**Pull request by bramtechs**
## Description 
Updates invalid querySelectors of the my.brain.fm page, so main functionality works again.
Resolves #7373

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image

</details>

### CODE REVIEW

File: websites/B/Brain.fm/presence.ts [hunk 1]
```diff
@@ -67,7 +69,7 @@ presence.on("UpdateData", () => {
 		else if (["/focus", "/relax", "/sleep"].includes(pathname)) {
 			const activity = document
 				.querySelector(
-					"#root > div > div > div.sc-sCrnh.fGBZrg > div.sc-gJLCiq.iGquue > div.sc-kBizdN.gGaVww > div.sc-jNpQCG.dAEozB > div > div > span"
+					"#root > div.sc-inJWWq.lbdOVj > div > div.sc-bImIVK.gtfUIb > div.sc-dZpgSj.bzGlhX > div.sc-jDIYFk.fbXWvM > div.sc-bdytox.kExulF > div > div > span"
 				)
```
Comment (DarkVillager) [inline websites/B/Brain.fm/presence.ts:73]: @theusaf how about the following instead?

It works with XPath :)
```ts
document.evaluate('//span[text()="Activity"]', document)?.iterateNext()?.parentElement?.querySelectorAll("span")?.[1]?.textContent
```

Comment (bramtechs) [reply websites/B/Brain.fm/presence.ts:73]: Seems interesting, this could prevent future breakage from class names changing.

Comment (theusaf) [reply websites/B/Brain.fm/presence.ts:73]: I think it's a good idea.

Comment (bramtechs) [reply websites/B/Brain.fm/presence.ts]: I managed to clean up some more searches by using their 'test-id' attributes. 
Personally I'm not really experienced with using XPath, so I'm using regular querySelectors for now.

**Review by theusaf — APPROVED** [review]
LGTM

**Review by Bas950 — APPROVED** [review]

**Pull request merged**